dynamic_rest/routers.py

```diff
@@ -179,6 +181,7 @@ def register_resource(self, viewset, namespace=None):
             'path': base_path,
             'viewset': viewset
         }
+        resource_name_map[resource_name] = resource_key
```

Comment:
Can we add a check to throw an error if `resource_name` is already there? Probably better to err on the side of not allowing ambiguity.